app/hotels/src/singleHotel/roomList/RoomRow.js

```diff
@@ -75,17 +74,14 @@ export class RoomRow extends React.Component<Props> {
 
   render = () => {
     const availableRoom = this.props.availableRoom;
-    const thumbnailUrl = idx(
-      availableRoom,
-      _ => _.room.roomPhotos[0].lowResUrl,
-    );
-    const amount = idx(availableRoom, _ => _.minimalPrice.amount) || null;
-    const currency = idx(availableRoom, _ => _.minimalPrice.currency) || null;
-    const selectableCount =
-      idx(availableRoom, _ => _.incrementalPrice.length) || 0;
-    const id = idx(availableRoom, _ => _.id) || '';
-    const selectedCount = idx(this.props.selected, _ => _[id]) || 0;
-    const room = idx(availableRoom, _ => _.room);
+    const thumbnailUrl = availableRoom?.room?.roomPhotos?.[0]?.highResUrl;
```

Comment:
thumbnail is given in high resolution: is that intended?

Reply:
yes, testers complain about thumbNailUrl to be to bad quality, I upgraded to lowResUrl which was fine, however stay22 only provides thumbNail and highRes url

Reply:
Could we document this somewhere?

Reply:
I will check with the provider if they can add lowResUrl also, so hopefully we can change it back. 

another possible approach is to use lowResUrl, and fallback to highResUrl if lowResUrl is null.